schema.yml

```diff
@@ -202,6 +204,51 @@ paths:
       security:
         - podcast_auth:
             - read:subscriptions
+  /episodes:
```

Comment:
@Sporiff Could I ask you to review? I tried to follow the format of the existing file :-)